<launch>
  <node type="visual_skin_node2" pkg="ay_vision" name="visual_skin_node_ay13l" output="screen">
    <param name="pkg_dir" value="$(find ay_vision)/" />
    <param name="cam_config" value="config/usbcam2fay13a_l.yaml" />
    <param name="stereo_config" value="config/usbcam2fay13a_l.yaml" />
    <param name="blobtrack_config" value="config/usbcam2fay13a_l.yaml" />
    <param name="objdettrack_config" value="config/usbcam2fay13a_l.yaml" />
    <param name="blob_calib_prefix" value="data_gen/blob_" />
    <param name="frame_skip" value="0" />
    <param name="target_fps" value="30" />
  </node>
  <node type="visual_skin_node2" pkg="ay_vision" name="visual_skin_node_ay13r" output="screen">
    <param name="pkg_dir" value="$(find ay_vision)/" />
    <param name="cam_config" value="config/usbcam2fay13a_r.yaml" />
    <param name="stereo_config" value="config/usbcam2fay13a_r.yaml" />
    <param name="blobtrack_config" value="config/usbcam2fay13a_r.yaml" />
    <param name="objdettrack_config" value="config/usbcam2fay13a_r.yaml" />
    <param name="blob_calib_prefix" value="data_gen/blob_" />
    <param name="frame_skip" value="0" />
    <param name="target_fps" value="30" />
  </node>
</launch>
